### PR TITLE
fix: repair helpers to work on newer versions

### PR DESCRIPTION
**Pull request by rxri**
~~Fixing home config is not feasible because it requires multiple regexes to patch home~v2 thus we decided to remove it. It was broken for very long time anyway, and I'm 99% sure it doesn't work for versions we support~~

### CODE REVIEW

**Comment by rxri** [comment]
* **#2888** <a href="https://app.graphite.dev/github/pr/spicetify/spicetify-cli/2888?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rxri and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/apply/apply.go [hunk 1]
```diff
@@ -361,22 +348,6 @@ func insertCustomApp(jsPath string, flags Flag) {
...
 
-func insertHomeConfig(jsPath string, flags Flag) {
-	if !flags.HomeConfig {
-		return
-	}
-
-	utils.ModifyFile(jsPath, func(content string) string {
-		utils.ReplaceOnce(
-			&content,
-			`([\w$_\.]+\.sections\.items)(\.map)`,
```
Comment (afonsojramos) [inline src/apply/apply.go:372]: Is this the only regex for home config?

Comment (rxri) [reply src/apply/apply.go:372]: yes

Comment (afonsojramos) [reply src/apply/apply.go:372]: So, what do you mean when you say "it requires multiple regexes to patch home~v2"?

Comment (rxri) [reply src/apply/apply.go:372]: it would require multiple regexes to work on the versions we support

Comment (afonsojramos) [reply src/apply/apply.go:372]: I'd be more comfortable in marking this as help needed to see if anyone can help than simply removing the feature I think

Comment (rxri) [reply src/apply/apply.go:372]: well, okay... just saying that no one uses it i think because only one person in last few months asked about it 🤷‍♀️

**Review by theRealPadster — APPROVED** [review]

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**